content/en/blog/_posts/2023-03-08-memory-qos-cgroups-v2/index.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: blog
+title: 'Quality-of-Service for Memory Resources'
+date: 2021-11-26
+slug: qos-memory-resources
+---
+
+**Authors:** Tim Xu (Tencent Cloud)
+
+Kubernetes v1.27, released in April 2023, introduced changes to an alpha feature, Memory QoS, that improves how Linux nodes implement memory resource requests and limits. Support for Memory QoS was initially added in Kubernetes v1.22, and later some [limitations](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2570-memory-qos#reasons-for-changing-the-formula-of-memoryhigh-calculation-in-alpha-v127) around the formula for calculating memory.high were identified. Those limitations are addressed in Kubernetes v1.27.
```

Comment:
Please always **manually wrap** these long lines for 1) ease of review; 2) ease of change tracking, especially for the downstream localization teams.

Reply:
Example:

```suggestion
Kubernetes v1.27, released in April 2023, introduced changes to an alpha feature,
Memory QoS, that improves how Linux nodes implement memory resource
requests and limits. Support for Memory QoS was initially added in Kubernetes v1.22,
and later some [limitations](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2570-memory-qos#reasons-for-changing-the-formula-of-memoryhigh-calculation-in-alpha-v127
around the formula for calculating memory.high were identified.
Those limitations are addressed in Kubernetes v1.27.
```